src/index.js

```diff
@@ -1,248 +1,70 @@
 const once = require("just-once");
-const { encode, decode } = require("isomorphic-textencoder");
-const debounce = require("just-debounce-it");
 
-const path = require("./path.js");
-const Stat = require("./Stat.js");
-const CacheFS = require("./CacheFS.js");
-const { ENOENT, ENOTEMPTY } = require("./errors.js");
-const IdbBackend = require("./IdbBackend.js");
-const HttpBackend = require("./HttpBackend.js")
-const clock = require("./clock.js");
+const PromisifiedFS = require('./PromisifiedFS');
+
+function wrapCallback (opts, cb) {
+  if (typeof opts === "function") {
+    cb = opts;
+  }
+  const _cb = cb;
+  cb = once((...args) => {
```

Comment:
`cb = once(_cb)`

Reply:
huh nice! I'm not doing ten things in that callback anymore since I've got a different `_wrap` to handle that stuff :)